### DISCUSSION TITLE
Does anyone else have a sound noise now when CTRL alt D?

### DISCUSSION THREAD

**Discussion by praytheturtle** [SnosMe/awakened-poe-trade#1189] [Q&A]
If you do please tell me how to turn it off, I've searched settings and nothing :( point me in the direction pwease

**Comment by SnosMe** [comment]
Someone said it can be Save Clip feature from Discord Beta.
Anyway some app on your PC is making sound, not awakened.

**Reply by Mulldawg90** [reply]
I was having the same issue, but its a strange interaction. The discord clip feature is (for me) ALT C, and Awakened is Control D. When I have Awakened off, only ALT C works to clip, but when I have Awakened on, Control D clips as well. As soon as I shut Awakened off, Control D doesn't clip any more. I am not sure why or how that would happen. Thank you so much for the help though.

**Reply by Mostafa0Yasser** [reply]
it was discord for me, thank you, when i closed it the sound was gone, now i will go through discord to see whats that

**Comment by allstries** [comment]
I have the same problem, it happened last season as well. When I press Ctrl + D, it will record the video automatically in my C drive.

**Comment by praytheturtle** [comment]
It’s when you stream on discord. Something with the clipping settingsSent from my iPhoneOn Oct 31, 2025, at 11:58 PM, Mostafa0Yasser ***@***.***> wrote:﻿
it was discord for me, thank you, when i closed it the sound was gone, now i will go through discord to see whats that
